easybuild/main.py

```diff
@@ -600,9 +603,11 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
     if '_' not in os.environ:
         os.environ['_'] = sys.executable
 
+    if any([args, logfile, testing]):
+        eb_go, cfg_settings = set_up_configuration(args=args, logfile=logfile, testing=testing)
+
     # purposely session state very early, to avoid modules loaded by EasyBuild meddling in
     init_session_state = session_state()
```

Comment:
move this up, do this *before* calling `set_up_configuration` as before (that may be important because `set_up_configuration` already changes stuff in the environment, like `$TMPDIR` for example)